### DISCUSSION TITLE
HM-600 turns on after manuell turn off, if DC input is on battery

### DISCUSSION THREAD

**Discussion by helgeerbe** [tbnobody/OpenDTU#559] [Q&A]
Hi, 

I'm not sure if this is an expected behavior. 

My HM-600 is synchronized to the grid. As input DC I use a 24 V battery. If input voltage drops below a given threshold (< 24 V), I turn the HM-600 off. 

OpenDTU live view becomes yellow and the AC side is turned off. Home Assistant tells me "Producing Off" and "Reachable On". Which is so far the expected behavior. In the event log I see "124 | Shut down by remote control".

But after a while the HM-600 starts producing. Live view becomes blue and I see "Producing ON" but no further entry in the event log.

As a workaround I must switch off AC output to the grid to ensure the HM-600 keeps off.

I made the following observations.

| Status | Action | Result |
| --- | --- | --- |
| grid off, HM - 600 off, DC input < 23.5 V | grid on | HM-600 starts producing |
| grid off, HM - 600 off, DC input < 23.5 V | do nothing | HM - 600 keeps off, but is reachable |
| grid on, HM - 600 off, DC input < 23.5 V | do nothing | HM - 600 starts producing after a while (within less a minute) |

Where could I find the reason for turning on?

**Comment by tbnobody** [comment]
> Live view becomes blue and I see "Producing ON" but no further entry in the event log.

But you still have "old" entries in the event log or is it empty?

**Comment by helgeerbe** [comment]
Eventlog from today. 
At 9:09 Inverter started, because battery was charged enough by MPPT. But inverter was not attached to the grid.

At 10:47 I attached the HM-600 to the grid. I stopped producing 4 times. But the inverter autmattically starts producing. 

So finally I removed the inverter from the grid, to load the battery.
![image](https://user-images.githubusercontent.com/59169507/216975031-45cddfa0-2b27-407d-a88b-ae615b153104.png)

**Comment by helgeerbe** [comment]
OK, some further observation. Above 24.11 V DC input turning the HM-600 off is stable.
When input voltage drops below 24.11 V the HMS-600 starts producing.

This is strange. HM-600 data sheet shows start voltage 22V and operation range is 16-60V?

Is it just my inverter? Can anybody confirm this?

**Comment by helgeerbe** [comment]
I replaced the HM-600 with a HM-300. Even with 22.3V DC input the inverter stays off as expected. 
Unfortunately I don't have an other HM-600 to do further test. I would be very interested, if somebody can do further low voltage tests.
